modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C

```diff
@@ -58,20 +59,18 @@ MultiSmoothSuperellipsoidIC::MultiSmoothSuperellipsoidIC(const InputParameters &
 void
```

Comment:
In the class parameters above here there is still a `bool` for `_prevent_overlap`, how the code is currently set up it has no function as the `ellipsoidsOverlap` function is always utilized. I had left it as an option because if a large enough `bubspac` is used this check becomes redundant. However for my purposes I will always use `ellipsoidsOverlap` anyway so this change won't affect my work.